Support/Testing/Travis/script.sh

```diff
@@ -71,6 +71,8 @@ check_dirty() {
     for f in "${dirty[@]}"; do
       echo "dirty: $f"
     done
+    echo "Here is the diff:"
+    git diff | cat
```

Comment:
Yeah it's in `man git` instead of `man git-diff`. That's probably why.

On Tue, May 3, 2016 at 12:05 PM, kokarez <notifications@github.com> wrote:

> In Support/Testing/Travis/script.sh
> <https://github.com/facebook/ds2/pull/312#discussion_r61937488>:
>
> > @@ -71,6 +71,8 @@ check_dirty() {
> >      for f in "${dirty[@]}"; do
> >        echo "dirty: $f"
> >      done
> > +    echo "Here is the diff:"
> > +    git diff | cat
>
> :D yea, didn't found it when I was in the man. Will do
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/312/files/e61e88015be046eac9a4a8c5946fe65bb1f9960b#r61937488>
>



-- 
Stephane Sezer